batch/batch/server/server.py

```diff
@@ -417,7 +417,39 @@ def to_json(self):
 log.info(f'app.root_path = {app.root_path}')
 
 
+authorized_users = [
+    'konradk@broadinstitute.org',
```

Comment:
I think auth0 user ids are prefixed with the provider but I forget the details.  @akotlar?

Reply:
auth0 ids are prefixed as you say, except `google-oauth2|id` instead of `google|id`: https://auth0.com/docs/api-auth/tutorials/adoption/scope-custom-claims
- some more detail: https://community.auth0.com/t/sub-claim-format-kubernetes-restricted-set/21996